tests/spec/core/pluralize-spec.js

```diff
@@ -189,6 +189,35 @@ describe("Core - Pluralize", () => {
     ).toBeTruthy();
   });
 
+  it("doesn't mishandle <dfn> referring plural nouns without singulars", async () => {
```

Comment:
I think we don't need this test as we don't need to test what `pluralize` package gives us. 
Unless `scissors` and `trousers` are names of potential APIs.

Reply:
Lets remove this test

Reply:
done. thanks.